client_code/Pivot/__init__.py

```diff
@@ -0,0 +1,59 @@
+# SPDX-License-Identifier: MIT
+#
+# Copyright (c) 2021 The Anvil Extras project team members listed at
+# https://github.com/anvilistas/anvil-extras/graphs/contributors
+#
+# This software is published at https://github.com/anvilistas/anvil-extras
+import anvil
+import anvil.js
+
+from ..utils import _component_helpers as helpers
+from ._anvil_designer import PivotTemplate
+
+__version__ = "1.6.0"
+
+pivottable_version = "2.23.0"
+jqueryui_version = "1.11.4"
+
+prefix = "https://cdnjs.cloudflare.com/ajax/libs"
+dependencies = [
+    f"{prefix}/pivottable/{pivottable_version}/pivot.min.css",
+    f"{prefix}/jqueryui/{jqueryui_version}/jquery-ui.min.js",
```

Comment:
I guess it could also be worth having a separate check for a clash with jquery-ui.
I can see someone else adding this in their native libraries for a different component?